tracer/src/Datadog.Trace/ClrProfiler/AutoInstrumentation/GraphQL/HotChocolate/IExecutionResultV13.cs

```diff
@@ -0,0 +1,17 @@
+// <copyright file="IExecutionResultV13.cs" company="Datadog">
+// Unless explicitly stated otherwise all files in this repository are licensed under the Apache 2 License.
+// This product includes software developed at Datadog (https://www.datadoghq.com/). Copyright 2017 Datadog, Inc.
+// </copyright>
+
+using System.Collections;
+using System.Collections.Generic;
+
+namespace Datadog.Trace.ClrProfiler.AutoInstrumentation.GraphQL.HotChocolate
+{
+    /// <summary>
+    /// HotChocolate.Execution.IExecutionResult interface for ducktyping
+    /// </summary>
+    internal interface IExecutionResultV13
```

Comment:
Well, even if the interface is empty, I need this object because I cast it to a IQueryResult (to get the Errors object that have been moved there).
I can't use the old IExecutionResult because the AutoInstrumentation failed to ducktype it on the newer version (some fields are not there anymore).

Reply:
Oh I thought it wasn't used, I searched for it in the PR and haven't found it. But maybe I had closed some files.
Thanks